src/WebJobs.Extensions.DurableTask/EndToEndTraceHelper.cs

```diff
@@ -726,6 +726,22 @@ public void ExtensionWarningEvent(string hubName, string functionName, string in
             }
         }
 
+        public void TraceConfiguration(
+            string hubName,
+            string configurationJsonString)
+        {
+            EtwEventSource.Instance.ExtensionConfiguration(
+                hubName,
+                LocalAppName,
+                LocalSlotName,
+                configurationJsonString,
+                ExtensionVersion);
+
+            this.logger.LogInformation(
+                "Durable extension configuration loaded: {details}. HubName: {hubName}. AppName: {appName}. SlotName: {slotName}. ExtensionVersion: {extensionVersion}.",
+                configurationJsonString, hubName, LocalAppName, LocalSlotName, ExtensionVersion);
+        }
```

Comment:
I created a new event specifically for tracing configuration to make it easier to search just for these events in Kusto.